test/homepage-noweb-test.js

```diff
@@ -68,6 +69,25 @@ suite.addBatch({
                 assert.ifError(err);
                 br.assert.success();
                 br.assert.text("H1", "pump.io API");
+            },
+            "it not has login or register buttons": function(err, br) {
+                br.assert.success();
+                br.assert.evaluate('document.getElementById("login")', null);
+                br.assert.evaluate('document.getElementById("register")', null);
+            }
+        },
+        "and we request javascript file": {
```

Comment:
> and we request a JavaScript file

:)

Reply:
This should be documented in [HACKING.md](https://github.com/pump-io/pump.io/blob/master/HACKING.md)